package.json

```diff
@@ -14,6 +14,9 @@
     "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/CATcher/",
     "ng:serve": "ng serve",
     "ng:serve:web": "npm run postinstall:web && ng serve -o",
+    "ng:serve:dev": "ng serve --configuration=dev -o",
+    "ng:serve:test": "ng serve --configuration=test -o",
+    "ng:serve:prod": "ng serve --configuration=production -o",
```

Comment:
Clarification: why do we need the `ng:serve:prod` command ?
We won't be using the development server for production mode yea? (`ng serve` uses Angular's dev server)

Reply:
Yes I initially used to it to test a bug with changing configurations.

Also the `dev` environment is redundant yes? I noticed some boilerplate code in the `angular.json` file defining a `dev` architecture for the angular build but most of our commands don't use this mode. There also isn't a `environment.dev.ts` 😅
Since I am making edits in angular.json thought I could just remove the `dev` build instructions, what do you think?

Reply:
Yea, `dev` isn't used but let's remove it in a separate PR